### ISSUE TITLE
[Tooltip] Linking target and tooltip (id/aria-describedby)

### ISSUE THREAD

**Issue by cpof-tea** [epam/UUI#1192]
## Now

Actual tooltip of `<Tooltip />` component has `role="tooltip"` and `aria-hidden="true"`. But it is not enough for accessibility tools. 

Target and tooltip should be linked by `id="ref"` on tooltip and by `aria-describedby="ref"` on target[^1]. We can pass to target whatever we want, but there's no way to pass any HTML attributes to tooltip element as for now.

### Workaround

As for now - rendering separate visually hidden description container in addition to `<Tooltip />`.

```jsx
<Tooltip ... >
  <Button
    rawProps={{ 'aria-describedby': 'ref' }} 
    ... 
  />
</Tooltip>
<div 
  id="ref" 
  role="tooltip" 
  aria-hidden="true"
>
  ...
</div>
```

## To Do

Ability to link target and tooltip. 

## Describe the solution you'd like

`rawProps` on `<Tooltip />`

Ability to pass `rawProps` to tooltip element.

```jsx
<Tooltip
  rawProps={{ id: 'ref' }}
  ...
>
  <Button
    rawProps={{ 'aria-describedby': 'ref' }}
    ...
  />
</Tooltip>
```

I would suggest to not stick to some kind of auto-linking solution (ex. `id=ref`), since tooltips have so many problems[^2] and not all tooltip-cases can be accessible with `aria-describedby` (ex. toogletips[^3]).

[^1]: [WAI-ARIA 1.2 #tooltip](https://www.w3.org/TR/wai-aria-1.2/#tooltip).
[^2]: [Tooltips in the time of WCAG 2.1](https://sarahmhigley.com/writing/tooltips-in-wcag-21/)
[^3]: Toggletips are like tooltips in the sense that they can provide supplementary or clarifying information. However, they differ by making the control itself supplementary: toggletips exist to reveal information balloons, and serve no other purpose. ([source](https://inclusive-components.design/tooltips-toggletips/#inclusivetoggletips))

**Comment by NatalliaAlieva** [comment]
Released in 5.0.0 ver.
